dist/pom.xml

```diff
@@ -723,7 +723,7 @@
           <configurationDirectory>config</configurationDirectory>
           <copyConfigurationDirectory>true</copyConfigurationDirectory>
           <extraJvmArguments>-XX:+ExitOnOutOfMemoryError
-            -Dfile.encoding=UTF-8 -Xshare:auto</extraJvmArguments>
+            -Dfile.encoding=UTF-8 -Xshare:auto -Djava.library.path="$BASEDIR/native-libs"</extraJvmArguments>
```

Comment:
:x: Instead of modifying the whole library path which might raise some eyebrows, I'd suggest to just tell RocksDB where we expect this library to be. The `RocksDB.loadLibrary()` call in `ZeebeRocksDbFactory` can take a path, let's use that instead.

Reply:
Tried using `RocksDB.loadLibrary("$BASEDIR/native-libs")`, but I think there is a bug in the RocksDB function that loads the native library, because it appends "jni" at the end of the filename twice:
```java.lang.UnsatisfiedLinkError: Can't load library: [redacted]/camunda/dist/target/camunda-zeebe-8.8.0-SNAPSHOT/native-libs/librocksdbjnijni-linux64.so``` 

The function is slightly different when no arguments are passed :shrug:

Reply:
Weird, but since we control how the file gets there, why not just rename it to the appropriate name? I mean, we should also fix it upstream, it's such an easy fix, but in the meantime we can just adapt to it?

Reply:
are you suggesting we can save the files as "librocksdbjnijni-{arch}" instead of the correct "librocksdbjni-{arch}"?

Reply:
Other than it looks funky, I don't see a huge issue